br-webui/views/mavproxy.liquid

```diff
@@ -0,0 +1,129 @@
+{% include ../_includes/header.liquid %}
+
+<script>
+var socket = io();
+
+const MAVPROXY_PARAM_FILE = 'mavproxy.param';
+
+var mavproxy_params_loaded = false;
+
+function saveOptions() {
+    var mavOptions = document.getElementById('mavOptions').value;
+    socket.emit('save params', {
+        'file':MAVPROXY_PARAM_FILE,
+        'params':mavOptions
+    });
+
+    // Disable inputs
+    document.getElementById('mavOptions').disabled      = true;
+    document.getElementById('mavRestartBtn').disabled   = true;
+    document.getElementById('saveOptionsBtn').disabled  = true;
+    document.getElementById('resetOptionsBtn').disabled = true;
+}
+
+function loadOptions() {
+    socket.emit('load params', {'file':MAVPROXY_PARAM_FILE});
+}
+
+function resetOptions() {
+    socket.emit('delete params', {'file':MAVPROXY_PARAM_FILE});
+
+    // Disable inputs
+    document.getElementById('mavOptions').disabled      = true;
+    document.getElementById('mavRestartBtn').disabled   = true;
+    document.getElementById('saveOptionsBtn').disabled  = true;
+    document.getElementById('resetOptionsBtn').disabled = true;
+}
+
+socket.on('save params response', function(data) {
+    // Enable inputs
+    document.getElementById('mavOptions').disabled      = false;
+    document.getElementById('mavRestartBtn').disabled   = false;
+    document.getElementById('saveOptionsBtn').disabled  = false;
+    document.getElementById('resetOptionsBtn').disabled = false;
+});
+
+socket.on('load params response', function(data) {
+    document.getElementById('mavOptions').value = data.params;
+    cam_params_loaded = true;
+
+    // Enable inputs
+    document.getElementById('mavOptions').disabled     = false;
+    document.getElementById('mavRestartBtn').disabled  = false;
+    document.getElementById('saveOptionsBtn').disabled = false;
+console.log(data.file.toString());
+    if (data.file.toString().indexOf("default") == -1) {
+        // Only enable restore default button if the loaded file is not default
+        document.getElementById('resetOptionsBtn').disabled = false;
+    }
+});
+
+socket.on('delete params response', function(data) {
```

Comment:
Should re-load parameters on webpage

Reply:
I think we are still missing this functionality.